src/main/java/uk/gov/hmcts/reform/civil/utils/DocmosisTemplateDataUtils.java

```diff
@@ -55,17 +59,19 @@ public static String fetchRespondentName(CaseData caseData) {
     public static String fetchApplicantName(CaseData caseData) {
         StringBuilder applicantNameBuilder = new StringBuilder();
 
-        if (caseData.getApplicant2() != null) {
+        if (caseData.getApplicant1() != null && caseData.getApplicant2() != null) {
             applicantNameBuilder.append("1 ");
             applicantNameBuilder.append(caseData.getApplicant1().getPartyName());
             soleTraderCompany(caseData.getApplicant1(), applicantNameBuilder);
             applicantNameBuilder.append(" & 2 ");
             applicantNameBuilder.append(caseData.getApplicant2().getPartyName());
             soleTraderCompany(caseData.getApplicant2(), applicantNameBuilder);
-        } else {
+        } else if (caseData.getApplicant1() != null) {
             applicantNameBuilder.append(caseData.getApplicant1().getPartyName());
             soleTraderCompany(caseData.getApplicant1(), applicantNameBuilder);
             litigationFriend(caseData.getApplicant1LitigationFriend(), applicantNameBuilder);
+        } else {
+            LOG.info("Applicant1 not found for claim number: " + caseData.getCcdCaseReference());
```

Comment:
if there is no applicant1 recorded on case data, that appears to be an invalid state of case data. shouldn't we throw the error here instead logging an info message ?

Reply:
Hi I add this Logging info message after discuss it with @indu2019 .

Reply:
This method is preparing name for the docmosis template, if no name is prepared due to invalid case data, we are passing null/empty name to template. I don't think that will be a valid scenario to have no name on official generated document.